---
title: "Supporting Information"
subtitle: "Effectiveness of antifungal treatments during chytridiomycosis epizootics in populations of an endangered frog"
author: 
  - Roland A. Knapp
  - Maxwell B. Joseph
  - Thomas C. Smith 
  - Ericka E. Hegeman
  - Vance T. Vredenburg
  - James E. Erdman, Jr.
  - Daniel M. Boiano
  - Andrea J. Jani
  - Cheryl J. Briggs
output: 
  bookdown::pdf_document2:
    keep_tex: TRUE
    toc: FALSE
bibliography: bibliography_appendix.bib
fontsize: 12pt
csl: ecology.csl
header-includes:
  - \usepackage{blkarray}
  - \usepackage{amsmath}
  - \newcommand{\matr}[1]{\mathbf{#1}}
  - \usepackage{setspace}\doublespacing
  - \usepackage[document]{ragged2e}
  - \usepackage{lineno}
  - \usepackage[none]{hyphenat}
editor_options: 
  chunk_output_type: console
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
options(knitr.kable.NA = '')
library(readr)
library(kableExtra)
```

\large PeerJ

\linenumbers

# Author Contributions {-}

Roland Knapp obtained the permits for all treatments, designed and implemented all of the treatments except at Treasure, conducted post-treatment surveys, managed the datasets, conducted final analyses of all of the treatment datasets except the capture-mark-recapture (CMR) dataset from the LeConte and Treasure treatments, and wrote most of the manuscript. 

Maxwell Joseph assisted with the Barrett-2009 and Dusy-2010 treatments and associated post-treatment surveys, analyzed the CMR dataset from the LeConte experiment and wrote the associated text, provided input on most of the analyses, and reviewed all drafts of the manuscript. 

Thomas Smith assisted with the LeConte and Treasure treatments, conducted post-treatment surveys for multiple treatments, analyzed the results of the Treasure treatment and wrote some of the associated text, conducted preliminary analyses of the Barrett-2009, Dusy-2010, and Dusy-2012 treatments, and reviewed all drafts of the manuscript. 

Ericka Hegeman assisted with the Treasure treatment, conducted post-treatment surveys for multiple treatments, managed the datasets, conducted preliminary analyses of the Barrett-2009 and Dusy-2010 treatments and wrote some of the associated text, and reviewed all drafts of the manuscript. 

Vance Vredenburg assisted with the design and implementation of the Dusy-2012 treatment, including supporting the development of the *J. lividum* qPCR assay, culturing of the *J. lividum* used in the treatment, and qPCR analysis of *J. lividum* samples.  

James Erdman conducted pre- and post-treatment surveys at Treasure, implemented the Treasure treatment, and reviewed all drafts of the manuscript. 

Daniel Boiano assisted with the LeConte treatment, facilitated the National Park Service permitting of all treatments conducted in Sequoia and Kings Canyon National Parks, and reviewed all drafts of the manuscript.

Andrea Jani assisted with the LeConte treatment, and conducted the Dusy Basin zoospore pool study, provided the associated dataset, conducted preliminary analyses, and wrote some of the associated text, and reviewed all drafts of the manuscript. 

Cheryl Briggs provided modeling results that motivated the treatments, assisted with the design of treatments, and reviewed all drafts of the manuscript.  

<!-- Reorganize sections. All methods together and all results together, and methods and results for each treatment? -->

# Methods {-}

## Visual encounter surveys {-}

We counted *R. sierrae* of all life stages (adults: $\geq$ 40 mm snout-vent length (SVL); subadults: <&nbsp;40 mm; tadpoles) using diurnal visual encounter surveys (VES) of the entire water body shoreline and the first 100 m of inlet and outlet streams. 
VES is commonly used in studies of MYL frogs to estimate the abundance of all life stages present at a site [e.g., vredenburg2010dynamics; knapp2016large; @knapp2000nonnative]. Counts are highly repeatable [@knapp2000nonnative], but underestimate the number of animals present.

## Itraconazole treatment {-}

For all itraconazole treatments, we held animals assigned to the "treated" group in large mesh pens (2 m x 2 m x 0.75 m) for the duration of the multi-day treatment period. 
Holding animals in pens assured that each animal could be captured and treated on each day of the treatment period; this would not have been possible if animals needed to be recaptured from the ponds each day. 
Pens were anchored in the littoral zone of the study lakes, and contained shallow water and shoreline habitats for basking and deeper water habitat (up to 0.7 m) that frogs and tadpoles use at night (Figure S1). 
Animals assigned to the untreated "control" group were held in pens only 3-24 hr (to prevent resampling the same individuals) and then released back into the lake. 

Although it would have been ideal to hold animals from both the treated and control groups in pens for the duration of the treatment period, doing so could have produced spurious and misleading results. 
Bd transmission is expected to increase with frog density [@rachowicz2007quantifying], and holding untreated control animals in pens at relatively high density could therefore have increased their Bd loads and reduced survival more than would be expected for animals in the treated group that were given daily antifungal baths. 
This would have biased the outcome toward lower survival of control animals compared to treated animals even if the antifungal treatment itself had no effect on survival. 
Assuming that holding animals in pens for several days has some negative effect (due to increased Bd transmission even in treated frogs, and lack of feeding opportunities), if our study design caused biases they should be conservative, i.e., reducing the survival of treated animals relative to control animals. 
Including a second control group, in which were untreated animals were held for the entire treatment period, could have been informative, but would have substantially reduced the number of animals included in the existing treated and control groups, and potentially affected our ability to detect treatment effects. 

To conduct the antifungal treatments, on each day during the multi-day treatment period we transferred all animals in the treated group from pens to small plastic tubs that contained a dilute solution [1.5 mg L^‑1^; @garner2009using] of the antifungal drug itraconazole (trade name = Sporonox). 
We treated frogs in batches of approximately 50, and tadpoles in batches of approximately 100. 
The volume of itraconazole solution varied between 2 and 5 L, and was varied between batches based on the number and life stage of animals being treated to allow all individuals to submerge fully. 
After 10 minutes of itraconazole exposure, animals were transferred from the tubs back to the pens. 
To determine treatment effectiveness, we re-swabbed all animals or a subset (depending on the experiment) at the end of the treatment period. 
After the final treatment, we released all animals from the pens back into the study lakes.

## Estimating the zoospore pool from water samples  {-}

We sampled the zoospore pool in each of the five ponds included in the Dusy itraconazole treatment experiment (Table 1) before and after the treatments (July 23-25 and August 21-24, respectively).
Water samples (six per pond) were collected by filtering pond water through a 0.22 mm pore polyethersulfone filter (Sterivex-GP; Millipore), until the filter clogged. 
Filters were immediately amended with sucrose lysis buffer (40 mmol/l 1 EDTA, 50 mmol/l 1 Tris-HCl, 750 mmol/l sucrose, pH adjusted to 8.0). 
We extracted DNA using the DNEasy Blood and Tissue kit (Qiagen).
Bd concentrations in water samples (environmental "Bd loads") were quantified using qPCR [see @jani2017epidemic for details], and Bd load was normalized to a 1-liter sample volume. 
For each sample, we ran three technical replicates, each from an independent sample dilution and on an independent PCR run. 
To minimize PCR inhibition, we diluted DNA extracts 50-fold based on pilot tests using undiluted, ten-, fifty-, and one hundred-fold dilutions. 
Finally, we included 21 negative controls: nine no-template-controls (three per PCR plate) and 12 field-collected negative controls (six water samples collected from a pond where all frogs were Bd-negative, each with 2 technical replicates). 

The 21 negative controls yielded no false-positive PCR reactions. 
However, it is common for technical replicates of environmental DNA samples to have a high rate of false-negatives due to low quantities of target DNA or PCR inhibitors in samples [@mosher2018beyond], and this was the case in our study.
Despite the fact that all five study ponds contained relatively large numbers of early life stage *R. sierrae* characterized by high Bd loads, of the 180 total replicates (5 ponds x 6 water samples x 3 technical replicates x 2 time periods (before and after treatment)), 49% had a Bd load = 0. 
Because the five ponds in the experiment were clearly Bd-positive, we considered replicates with Bd load = 0 to be false negative replicates, which we excluded from the analysis of the effect of itraconazole treatment on the Bd zoospore pool. 
This resulted in one pond assigned to the treated group being dropped from the analysis due to a lack of any Bd-positive replicates. The effect of itraconazole treatment on Bd concentrations on filters was evaluated using the model *bd_load ~ (pre_post x treatment) + (1 | sample_id)* (family = negative binomial, pre_post = [before treatment, after treatment], treatment = [treated, control], sample_id included as a group-level effect to account for technical replicates). 


## Itraconazole treatment of adults

*Treasure Lake*. In 2018, we detected a Bd epizootic in he Treasure Lake *Rana sierrae* population (Table S1), providing another opportunity to test the effectiveness of antifungal treatment on adult frogs.
the treatment was conducted as a management action instead of an experiment because relatively few adults remained in this population at the time of the treatment. 
As such, dividing the frogs into treated and control groups would have provided little statistical power to detect between-group differences. 
Although the lack of an experimental design limits the generality of our findings, the treatment is included here because of the additional insights the results provide. 
Specifically, the greater range of treatment days to which frogs were exposed (compared to the LeConte treatments) provided an opportunity to evaluate the effectiveness of itraconazole treatment on Bd loads as a function of the number of daily treatments frogs received.

We used the same methods as described for the LeConte treatments, with two important differences: (1) all frogs were treated (there was no control group), and (2) new frogs were captured from the lake and added to the pens during the first five days of the 7-day treatment period. 
We captured and treated 28 frogs on July 16, then added and treated an additional 24, 7, 7, 4, and 4 frogs on July 17 through 21, respectively. 
Although we captured and treated a total of 74 frogs, we released only 33 live frogs at the end of the treatment due to chytridiomycosis-caused mortality throughout the treatment period.
In addition to swabs collected from all frogs immediately following their initial capture, we also collected swabs from each surviving frog after the final itraconazole treatment.


## *Janthinobacterium lividum* qPCR assay {-}

The *J. lividum* used to develop the real-time PCR assay was a strain provided by Reid Harris, Department of Biology, James Madison University. 
*J. lividum* DNA was extracted from a pure culture of this strain using a Qiagen DNeasy blood and tissue kit following the manufacturer's protocol. 
The DNA collected was amplified using methods in @harris2009skin. The PCR product was viewed on a 1% agarose gel, and the 500 bp product was sequenced. 

Primers were designed to be specific to *J. lividum* and sequences were: 

* Jliv_For3 ATGCCACCGACGGCTACCA
* Jliv_Rev1 ACGGCGGGATGGTCATCAC

The minor groove binder probe sequence was:

* JLIVT 6FAM AACATCGTTTGCTGTCCGTTGA MGBNFQ

After assay optimization, the 25 $\mu$L reaction volumes included 0.5 $\mu$L of each primer at a concentration of 10 $\mu$M, 0.375 $\mu$L of MGB probe at a concentration of 250 nM, 12.5 $\mu$L of Taqman MasterMix (Applied Biosystems), 1.25 $\mu$L BSA, and 5 $\mu$L of template. 
The amplification conditions consisted of an initial cycle of 2 min at 50&deg;C and 10 min at 95&deg;C, followed by 50 cycles at 95&deg;C for 15 s, 58&deg;C for 30 s, and 65&deg;C for 45 s. 
To create standards, DNA was extracted from pure cultures of *J. lividum* with an UltraClean microbial DNA isolation kit (MoBio), and diluted to 10^4^, 10^3^, 10^2^, and 10^1^ *J. lividum* genome equivalents. 
A standard curve was generated for each 96-well plate to estimate the number of *J. lividum* genome equivalents in sample extracts.

## Statistical analyses

### Simple and multilevel models using the brms package

When using the brms package, our analysis workflow included starting with a model that included all relevant population-level ("fixed") effects and their interactions ("full model"), and checking model fit using visualizations of leave-one-out ("LOO") probability integral transformations [@gelman2013bayesian; @vehtari2017practical]. 
When data contained possible hierarchical structure (which could result in non-independence), we added group-level ("random") effects to the full model. 
Bd load and frog counts were the most commonly modeled response variables, and models that used a negative binomial family generally produced the best fit. 
However, when appropriate, we also evaluated the fit of models that used other model families, including Poisson and zero-inflated negative binomial. 
We compared fits of models using LOO cross-validation and the *loo* package [@vehtari2017practical]. 
For all models, we used brms defaults for priors, number of chains (4), and warmup and post-warmup iterations (1000 for each). We evaluated the adequacy of posterior samples using trace plots, Gelman-Rubin statistics (Rhat), and measures of effective sample size ("bulk-ESS", "tail-ESS"). 
When using negative binomial models (most analyses), the Bd load data were rounded to integer values to produce count data. 

When necessary, we developed distributional models in which predictor terms are specified for other parameters of the response distribution instead of only the mean (e.g., negative binomial overdispersion ("shape"), zero-inflation ("zi"); see brms vignette, "Estimating distributional models with brms": https://paul-buerkner.github.io/brms/articles/brms_distreg.html). 
The overdispersion parameter $\phi$ controls the variance of the negative binomial distribution relative to the expected value $\mu$, such that the variance of the negative binomial distribution is $\mu + \mu^2 / \phi$. 
Modeling effects on overdispersion and zero-inflation can be important for improving model fit. 
For example, itraconazole treatment can reduce not only mean Bd load, but also the variation around the mean (i.e, overdispersion) and amount of zero-inflation. 
Improving model fit was our primary interest in using distributional models, and not gaining insights into the causes of overdispersion or zero-inflation. 
Therefore, when we used distributional models, we limit our descriptions of model results largely to effects of predictors on the mean. 

### Hidden Markov model for the LeConte treatment experiment {-}

We tracked the fates of individual animals in the LeConte population using a multi-state hidden Markov model. 
We consider three possible observations for each individual $i=1, ..., M$ in primary period $t=1, ..., T$, on secondary period $j=1, ..., J_t$, where $T$ is the total number of primary periods and $J_t$ is the number of secondary periods in primary period $t$: 

\begin{itemize}
    \item $y_{i, t, j}=1$ detected at the upper site
    \item $y_{i, t, j}=2$ detected at the lower site
    \item $y_{i, t, j}=3$ not detected
\end{itemize}

We use parameter-expanded data augmentation to account for the fact that the total number of adults in the population is unknown [@royle2012parameter]. 
Across the entire time period of the study, we assume $N_s$ unique individuals have been alive at either site. 
We observe $N$ unique individuals across all surveys, where $N \leq N_s$. 
An estimate of $N_s$ can be acquired by augmenting the observed capture histories with additional capture histories that consist entirely of non-detections, thus modeling a large number $M > N_s$ of individuals, $M - N_s$ of which never existed [@royle2009analysis]. 
Here, $M$ was chosen to be 2182 (1212 observed unique individuals plus 970 augmented individuals). 
We verified that posterior estimates of $N_s$ were much less than $M$ to avoid problems on the boundary of this augmented parameter space [@dennis2015computational]. 

We denote the true state of individual $i$ in primary period $t$ as $s_{i, t}$. 
We assume that within a primary period, the state of each individual does not change. 
This assumption is justified by the short time intervals between secondary periods within primary periods.
Four states are possible: 

\begin{itemize}
    \item $s_{i, t} = 1$ alive at the upper site
    \item $s_{i, t} = 2$ alive at the lower site
    \item $s_{i, t} = 3$ not recruited
    \item $s_{i, t} = 4$ dead
\end{itemize}

The "not recruited" state applies to individuals that have not yet entered the population, including individuals that haven't reached adulthood or pseudo-individuals created via data augmentation (as described above). 

*Observation model*. An emission matrix $\matr{\Omega}^{(t)}$ links observations to hidden states for primary period $t$.
The rows in $\matr{\Omega}^{(t)}$ correspond to the state of an individual in primary period $t$, and the columns correspond to observation probabilities such that the entry in the $m^{th}$ row, $n^{th}$ column is $Pr(y_{i, t, j} = n \mid s_{i, t} = m)$:

\[
\matr{\Omega}^{(t)} =
\begin{blockarray}{cccc}
  \text{Detected: upper} & \text{Detected: lower} & \text{Not detected}\\
\begin{block}{(ccc)c}
  p_t & 0 & 1 - p_t & \begin{tabular}{@{}c@{}}Alive: upper \end{tabular} \\
  0 & p_t & 1 - p_t & \begin{tabular}{@{}c@{}}Alive: lower \end{tabular} \\
  0 & 0 & 1 & \begin{tabular}{@{}c@{}}Not recruited \end{tabular} \\
  0 & 0 & 1 & \begin{tabular}{@{}c@{}}Dead \end{tabular} \\
\end{block}
\end{blockarray}
 \]

Here, $p_t$ is the probability of detection for an individual if it is alive in primary period $t$. 
We allowed detection probabilities to vary over time [@joseph2018]:

$$\text{logit}(p_t) = \alpha_0 + \epsilon^{(p)}_t,$$

where $\alpha_0$ is an intercept parameter, and $\epsilon^{(p)}_t$ is an adjustment on detection probability for primary period $t$.

*State model*. The hidden states of each individual evolve as a Markov process with transition matrix $\matr{\Psi}^{(i, t)}$, where the element in the $m^{th}$ row, $n^{th}$ column is $\text{Pr}(s_{i, t+1} = n \mid s_{i, t} = m)$:
\[
\matr{\Psi}^{(i, t)} =
\begin{blockarray}{ccccc}
  \text{Alive: upper} & \text{Alive: lower} & \text{Not recruited} & \text{Dead} \\
\begin{block}{(cccc)c}
  \phi_{i, t} (1 - \nu^{(l)}) & \phi_{i, t} \nu^{(l)} & 0 & 1 - \phi_{i, t} & \begin{tabular}{@{}c@{}} Alive: upper \end{tabular} \\
  \phi_{i, t} \nu^{(u)} & \phi_{i, t} (1 - \nu^{(u)}) & 0 & 1 - \phi_{i, t} & \begin{tabular}{@{}c@{}} Alive: lower \end{tabular} \\
  \gamma_t \rho^{(u)} & \gamma_t (1 - \rho^{(u)}) & 1 - \gamma_t & 0 & \begin{tabular}{@{}c@{}} Not recruited \end{tabular} \\
  0 & 0 & 0 & 1 & \begin{tabular}{@{}c@{}} Dead \end{tabular} \\
\end{block}
\end{blockarray}
 \]

Here, $\phi_{i, t}$ is the probability of survival, $\nu^{(l)}$ and $\nu^{(u)}$ are the probabilities of moving to the lower or upper site respectively (conditional on survival), $\gamma_t$ is the probability of recruitment, and $\rho^{(u)}$ is the probability of recruiting into the upper site  conditional on recruitment. 

Survival probabilities were modeled as a function of Bd load: 

$$\text{logit}(\phi_{i, t}) = \beta_0 + \beta_\text{g[i]}^{(g)} + \beta_\text{g[i]}^{(z)} z_{i, t},$$

where $\beta_0$ is an intercept parameter, $\beta_\text{g[i]}^{(g)}$ is an adjustment for group $g$, $g[i]$ is the group that individual $i$ belongs to, $\beta_\text{g[i]}^{(z)}$ is the effect of Bd load on group $g$, and $z_{i, t}$ is the Bd load of individual $i$ in primary period $t$.

We treat the field experiment in 2015 as the first primary period, for which states are known for experimental animals. 
For example, if individual $i$ was captured and released at the upper site, then we know that $s_{i, t} = 1$. 
Initial states are not known for non-experimental animals, which could have been alive at either site (states 1 or 2), or in the not recruited class (state 3). 
Note that we are only interested in modeling the state and capture histories of animals that might have been alive. 
We assigned a Dirichlet(1, 1, 1) prior for the initial state distribution for non-experimental frogs, which assigns equal prior density to each initial state.

*Bd load model*. We modeled Bd loads as being normally distributed on a transformed scale. 
Raw Bd loads were transformed using a $\log_{10}$ + 1 transformation, then centered and scaled to have mean zero and unit standard deviation (in an attempt to avoid ill-conditioning, as expected Bd load is used in the detection and survival model components).
Let $z^{\text{obs}}_{i, t}$ represent the observed transformed Bd load, and $z_{i, t}$ represent the expected value for individual $i$ on primary period $t$.
The observation model for Bd loads of detected individuals was Gaussian on the transformed scale: 

$$z^{\text{obs}}_{i, t} \sim \text{Normal}(z_{i, t}, \sigma),$$

\noindent where $\sigma$ is an observation-level standard deviation parameter. 

Expected Bd load was modeled as a function of treatment, primary period, and individual identity: 

$$z_{i, t} = \mu_t + \beta_{\text{trt}[i]} + \epsilon_i,$$

\noindent where $\mu_t$ is a time-varying intercept, $\beta_{\text{trt}[i]}$ is an adjustment for the treatment group of individual $i$ (denoted $\text{trt}[i])$ to account for differences in mean Bd loads between treated, control, and non-experimental animals, and $\epsilon_i$ is an individual-level adjustment.

*Prior distributions*. We expected movement among sites to be rare, so both movement parameters ($\nu^{(l)}$ and $\nu^{(u)}$) were assigned Beta(2, 20) priors.
Primary period and individual-level adjustments were modeled using zero-mean normal distributions with unknown standard deviations specific to the process of interest, e.g., for the probability of entering the population: $[\lambda_{1:T}] = \prod_{t = 1}^{T} \text{Normal}(\lambda_t | 0, \sigma^{(\lambda)})$. 
Standard deviation parameters were given unit scale half normal priors, and all remaining parameters were given unit scale normal priors.

*Inference*. We sampled from the posterior distribution of this model using dynamic Hamiltonian Monte Carlo in Stan. 
We drew 3000 iterations for each of four chains, using a maximum treedepth of 11 and an adapt_delta value of 0.99.
Convergence was checked by visual inspection of traceplots and with Rhat values, using Rhat < 1.1 as a threshold.
Models were fit using the rstan R package, version 2.21.2 [@rstan2020].

### Itraconazole treatment of adults

We assessed treatment effectiveness by comparing Bd loads measured before and after treatment, using the model *bd_load ~ trt_period* (family = negative binomial, trt_period = [begin, end]). 
To evaluate the effectiveness of itraconazole treatment on Bd loads as a function of the number of daily treatments frogs received, we calculated treatment effectiveness for individual frogs as the negative log ratio of pre-treatment to post-treatment Bd loads (hereafter, "LRR"): -log~10~((load~pre~ + 1)/(load~post~ + 1). 
Larger absolute values of LRR indicate a larger reduction in Bd load. 
To evaluate the factors influencing treatment effectiveness on individual frogs, we used the model *LRR ~ (capture_bdload_std x days_inside)* (family = gaussian, capture_bdload_std = Bd load prior to treatment standardized to mean = 0 and standard deviation = 1, days_inside = number of treatments a frog received). 


# Results {-}

## Estimating the zoospore pool from water samples  {-}

Before treatment, zoospore pools (measured as Bd load on collected filters) in the ponds assigned to the control and treated groups were similar (Figure S2). 
After treatment, zoospore pools in control ponds may have increased slightly, but remained relatively constant in treated ponds (Figure S2). 
Model results indicated that the estimated effects of treatment, basin, and the (treatment x basin) interaction term were all unimportant (Table S5). 
Therefore, assuming that the sampling method was adequate to accurately quantify pond-wide zoospore concentrations, the treatment of even a relatively large fraction of the resident *R. sierrae* in the study ponds did not measurably alter the zoospore pools. 
To avoid the high number of false-negative filters obtained using our methods, future studies attempting to quantify zoospore pools should consider using methods that allow filtering of larger volumes of water. 


## Itraconazole treatment of adults

*Leconte Basin*. 
```{r read-stan-results, include=FALSE}
pct_observed <- readr::read_csv(here::here("stan", "pct_observed.csv"))
load_diff <- readr::read_csv(here::here("stan", "load_diffs.csv")) %>%
  dplyr::mutate(diff = round(diff))
survival_diff <- readr::read_csv(here::here("stan", "survival_diffs.csv"))
abund_2018 <- readr::read_csv(here::here("stan", "2018_abund.csv"))
trt_2018 <- dplyr::filter(abund_2018, trt == "treated")
new_2018 <- dplyr::filter(abund_2018, trt == "new")
ctrl_abund <- readr::read_csv(here::here("stan", "ctrl_abund.csv"))
```

Based on CMR modeling, across the entire duration of the experiment (2015-2018), the 1206 unique individuals included in the study were estimated to represent approximately `r pct_observed$p[2]`% (posterior median) of the adults that existed in the LeConte population during this time (CI: `r pct_observed$p[1]`% -- `r pct_observed$p[3]`%). 
Between frog release in 2015 and the final survey in 2018, seven recaptured individuals moved between the two basins. 
All seven were in the treated group and moved from the upper to the lower basin. 
These individuals were included in counts of unique individuals in the basin in which they were captured.
In CMR surveys conducted during 2016-2018, a total of 2208 adult frogs were captured, representing 831 unique individuals. 
Of the 745 unique frogs captured in the lower basin, 132 were in the treated group, two were in the control group, and 611 were not part of the original treatment experiment (“non-experimental” frogs). 
In the upper basin, 89 unique frogs were captured, of which 81 were in the treated group and eight were non-experimental. 
No control frogs were captured in the upper basin. 

*Treasure Lake*. Similar to the situation in LeConte Basin, adult frog Bd loads were very high at Treasure Lake during early summer 2018, and at the start of the itraconazole treatment (Figure 5). 
Itraconazole treatment reduced Bd loads by more than two orders of magnitude (Figure 5; Bd loads on 2018-07-23 versus on days 2018-07-16 to 2018-07-21). 
Model results affirmed the important effect of treatment (Table S9). 

The number of itraconazole treatments a frog received ("days_inside") increased treatment effectiveness (Table S10). Initial Bd load ("capture_bdload_std") and the (days_inside x capture_bdload_std) interaction term were both unimportant (Table S10). 

Of the 33 frogs that were released back into the lake following treatment, 16 were recaptured in the CMR survey conducted one month later (Figure 5).
In addition, one non-experimental adult frog was captured, and one dead tagged (i.e., treated) adult was found. 
Bd loads of most recaptured frogs were low compared to those of frogs at the start and end of the treatment period (Figure 5, S3). 
There was no obvious relationship between the number of treatments a frog received and whether or not it was recaptured one month later (Figure S3). 
In surveys conducted in 2019 (the year following treatment) and 2020, we observed no *R. sierrae* of any life stage. 
Therefore, despite the substantial reduction in Bd loads caused by the 2018 treatment and the relatively large fraction of treated frogs recaptured one month later, few or no frogs survived overwinter until summer 2019. 


# Literature Cited {-}

<div id="refs"></div>

\clearpage

# Tables {-}

## Table S1 {-}

Characteristics of all sites used in the antifungal treatment experiments

```{r, include=FALSE}
t_1 <- read_csv(file = here::here("out", "tables", "expt_sites.csv"))
```
```{r tableS1, echo=FALSE}
kbl(
  t_1, col.names = c("Basin", "Site ID", "Experiment", "Life stage", "Category", "Elevation", "Depth", "Area"),
  booktabs = TRUE, linesep = "") %>%
  row_spec(0, bold = TRUE) %>%
  footnote(general = "Units for Elevation and Depth are meters and those for Area are m\\\\textsuperscript{2}", 
           general_title = "", escape = FALSE) %>%
  footnote(general = "Treatment = ``itracon + Jliv'' indicates the use of itraconazole and \\\\textit{J. lividum}", 
           general_title = "", escape = FALSE) %>%  
  footnote(general = "Life stage = ``early'' indicates tadpoles and subadults", general_title = "") %>%  
  kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S2 {-}

Timelines for antifungal treatment experiments conducted in the lower and upper basins of the LeConte study area in 2015.

(a) Lower basin
```{r, include=FALSE}
t_lower <- read_csv(file = here::here("out", "tables", "timeline-lower.csv")) 
```
```{r tableS2a, echo=FALSE}
kbl(t_lower, booktabs = TRUE, linesep = "") %>%
      row_spec(0, bold = TRUE) %>%
      column_spec(1, width = "7em") %>%
      column_spec(2, width = "7em") %>%
      column_spec(3, width = "7em") %>%
      column_spec(4, width = "7em") %>%
      column_spec(5, width = "7em") %>%
      column_spec(6, width = "7em") %>%
      column_spec(7, width = "7em") %>%
      kable_styling(latex_options = "scale_down")
```

(b) Upper basin
```{r, include=FALSE}
t_upper <- read_csv(file = here::here("out", "tables", "timeline-upper.csv")) 
```
```{r tableS2b, echo=FALSE}
kbl(t_upper, booktabs = TRUE, linesep = "") %>%
      row_spec(0, bold = TRUE) %>%
      column_spec(1, width = "7em") %>%
      column_spec(2, width = "7em") %>%
      column_spec(3, width = "7em") %>%
      column_spec(4, width = "7em") %>%
      column_spec(5, width = "7em") %>%
      column_spec(6, width = "7em") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S3 {-}

For the itraconazole treatment experiments in Barrett and Dusy basins, results of model comparing Bd loads on frogs in ponds assigned to control and treated groups immediately before the treatment period (model family is negative binomial). 

```{r, include=FALSE}
barrettdusy_m1 <- read_csv(file = here::here("out", "tables", "barrettdusy-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS3, echo=FALSE}
kbl(barrettdusy_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S4 {-}

For the itraconazole treatment experiments in Barrett and Dusy basins, results of model comparing Bd loads on frogs assigned to the treated group from before versus the end of the treatment period (model family is negative binomial). 

```{r, include=FALSE}
barrettdusy_m2c <- read_csv(file = here::here("out", "tables", "barrettdusy-m2c.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS4, echo=FALSE}
kbl(barrettdusy_m2c, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 9) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S5 {-}

For the Dusy Basin itraconazole treatment experiment, results of model comparing zoospore pools of ponds assigned to the control and treated groups before and after treatment (model family is negative binomial).   

```{r, include=FALSE}
dusy2010_m1 <- read_csv(file = here::here("out", "tables", "dusy2010-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS5, echo=FALSE}
kbl(dusy2010_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Group-level effects", 1, 1) %>%
      pack_rows("Population-level effects", 2, 5) %>%
      pack_rows("Family-specific parameters", 6, 6) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S6 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs assigned to the control and treated categories immediately before the treatment period (model family is negative binomial).   

```{r, include=FALSE}
leconte2015_m1 <- read_csv(file = here::here("out", "tables", "leconte2015-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS6, echo=FALSE}
kbl(leconte2015_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

## Table S7 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs assigned to the treated group from before versus the end of the treatment period (model family is negative binomial). 

```{r, include=FALSE}
leconte2015_m2 <- read_csv(file = here::here("out", "tables", "leconte2015-m2.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS7, echo=FALSE}
kbl(leconte2015_m2, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S8 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs in the treated group that survived versus died (model family is bernoulli).   

```{r, include=FALSE}
leconte2015_m4 <- read_csv(file = here::here("out", "tables", "leconte2015-m4.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS8, echo=FALSE}
kbl(leconte2015_m4, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

## Table S9 {-}

For the Treasure Lakes Basin itraconazole treatment, results of model comparing Bd loads on frogs before versus the end of the treatment period (model family is negative binomial). 

```{r, include=FALSE}
treasure_m1 <- read_csv(file = here::here("out", "tables", "treasure-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS9, echo=FALSE}
kbl(treasure_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 3) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S10 {-}

For the Treasure Lakes Basin itraconazole treatment, results of model evaluating predictors of treatment effectiveness (model family is gaussian).

```{r, include=FALSE}
treasure_m2 <- read_csv(file = here::here("out", "tables", "treasure-m2.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS10, echo=FALSE}
kbl(treasure_m2, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

## Table S11 {-}

For the Dusy Basin microbiome augmentation experiment, results of model comparing Bd loads on frogs assigned to the control and treated groups immediately before the itraconazole treatment period (model family is negative binomial).   
 
```{r, include=FALSE}
dusy2012_m1 <- read_csv(file = here::here("out", "tables", "dusy2012-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS11, echo=FALSE}
kbl(dusy2012_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 3) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S12 {-}

For the Dusy Basin microbiome augmentation experiment, results of model comparing Bd loads on frogs assigned to the treated category from immediately before versus at the end of the itraconazole treatment (model family is zero-inflated negative binomial).   

```{r, include=FALSE}
dusy2012_m2a <- read_csv(file = here::here("out", "tables", "dusy2012-m2a.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS12, echo=FALSE}
kbl(dusy2012_m2a, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 4) %>%
      row_spec(0, bold = TRUE) %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

# Figures {-}

## Figure S1 {-}

```{r figS1, echo=FALSE, fig.cap=""}
knitr::include_graphics(here::here("out", "figures", "dusy-2012-pens.jpg"))
```

Photograph showing mesh pens used to hold subadult *R. sierrae* during the 2012 microbiome augmentation experiment in Dusy Basin.  

\clearpage

## Figure S2 {-}

```{r figS2, echo=FALSE, fig.cap="", out.width="450px"}
knitr::include_graphics(here::here("out", "figures", "dusy-zsppool.png"))
```

In the Dusy Basin study ponds assigned to the control or treated groups, zoospore pools measured before and after itraconazole treatment of *R. sierrae*. 
The y-axis displays Bd load per water sample, normalized to a 1-liter sample volume. Each dot represents a single sample, and median values for each treatment period are indicated with a black diamond. 
The number of samples included is displayed above the x-axis. 

## Figure S3 {-}

<!-- Consider better facet labels (e.g., "Died during experiment", "Never recaptured', "Recaptured") -->

```{r figS3, fig.cap="", echo=FALSE}
knitr::include_graphics(here::here("out", "figures", "treasure-multicapture-loads.png"))
```

For all adult *R. sierrae* in the 2018 Treasure Lakes itraconazole treatment, Bd loads over a two month period that includes the July treatment (16 July to 23 July 2018) and August follow-up surveys. 
Points from the same frog are connected by a line. 
Panel labels are as follows: 1 = frog that died during the treatment ("non-survivor"), 2 = survivor that was not recaptured during the post-release survey in August, and 3 = survivor that was recaptured during the post-release survey. 
The single non-experimental frog captured in August was not included in the treatment (treatment duration = "NA") and is included in panel 1. 

\clearpage

