---
title: "Supporting Information"
subtitle: "Effectiveness of actions to mitigate impacts of the amphibian chytrid fungus on mountain yellow-legged frog populations"
author: "Roland A. Knapp, et al."
output: 
  bookdown::pdf_document2:
    keep_tex: TRUE
    toc: FALSE
bibliography: bibliography_appendix.bib
fontsize: 12pt
csl: ecology.csl
header-includes:
  - \usepackage{blkarray}
  - \usepackage{amsmath}
  - \newcommand{\matr}[1]{\mathbf{#1}}
  - \usepackage{setspace}\doublespacing
  - \usepackage[document]{ragged2e}
  - \usepackage{lineno}
  - \usepackage[none]{hyphenat}
editor_options: 
  chunk_output_type: console
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
options(knitr.kable.NA = '')
library(readr)
library(kableExtra)
```

\linenumbers

# Methods {-}

## Estimating the zoospore pool from water samples  {-}

Pond water was filtered through a 0.22 mm pore polyethersulfone filter (Sterivex-GP; Millipore), until the filter clogged. 
Filters were immediately amended with sucrose lysis buffer (40 mmol/l 1 EDTA, 50 mmol/l 1 Tris-HCl, 750 mmol/l sucrose, pH adjusted to 8.0). 
DNA was extracted using the DNEasy Blood and Tissue kit (Qiagen). 
DNA extracts were diluted 1:50 for qPCR. 
Quantitative PCR conditions and standards were as in [@jani2017epidemic], with a six-point standard curve. 
For each sample, we ran three technical replicates, each from an independent sample dilution and on an independent PCR run. 
It is common for technical replicates of environmental DNA samples to give variable results due to PCR inhibitors or low quantities of target DNA in samples [@mosher2018beyond]. 
We therefore conducted tests to establish criteria for calling samples Bd-positive or Bd-negative. 
We ran six negative controls (water filter samples from a Bd-naÃ¯ve pond) on two separate replicate PCR runs, for a  total of 12 PCR reactions, all of which returned Bd-negative data (0 false positive results out of 12 PCR reactions). 
We also ran no template controls in triplicate on every PCR run, with no false positives. 
Therefore, if a sample had at least one positive technical replicate we considered the sample to be positive and calculated the mean across technical replicates after dropping the presumed false negatives. 
To minimize PCR inhibition, we diluted DNA extracts fifty-fold based on pilot tests using undiluted, ten-, fifty-, and one hundred-fold dilutions. 

## *Janthinobacterium lividum* qPCR assay {-}

The *J. lividum* used to develop the real-time PCR assay was a strain provided by Reid Harris, Department of Biology, James Madison University. 
*J. lividum* DNA was extracted from a pure culture of this strain using a Qiagen DNeasy blood and tissue kit following the manufacturer's protocol. 
The DNA collected was amplified using methods in [@harris2009skin]. The PCR product was viewed on a 1% agarose gel, and the 500 bp product was sequenced. 

Primers were designed to be specific to *J. lividum* and sequences were: 

* Jliv_For3 ATGCCACCGACGGCTACCA
* Jliv_Rev1 ACGGCGGGATGGTCATCAC

The minor groove binder probe sequence was:

* JLIVT 6FAM AACATCGTTTGCTGTCCGTTGA MGBNFQ

After assay optimization, the 25 $\mu$L reaction volumes included 0.5 $\mu$L of each primer at a concentration of 10 $\mu$M, 0.375 $\mu$L of MGB probe at a concentration of 250 nM, 12.5 $\mu$L of Taqman MasterMix (Applied Biosystems), 1.25 $\mu$L BSA, and 5 $\mu$L of template. 
The amplification conditions consisted of an initial cycle of 2 min at 50&deg;C and 10 min at 95&deg;C, followed by 50 cycles at 95&deg;C for 15 s, 58&deg;C for 30 s, and 65&deg;C for 45 s. 
To create standards, DNA was extracted from pure cultures of *J. lividum* with an UltraClean microbial DNA isolation kit (MoBio), and diluted to 10^4^, 10^3^, 10^2^, and 10^1^ *J. lividum* genome equivalents. 
A standard curve was generated for each 96-well plate to estimate the number of *J. lividum* genome equivalents in sample extracts.

## Hidden Markov model for the LeConte treatment experiment {-}

We tracked the fates of individual animals in the LeConte population using a multi-state hidden Markov model. 
We consider three possible observations for each individual $i=1, ..., M$ in primary period $t=1, ..., T$, on secondary period $j=1, ..., J_t$, where $T$ is the total number of primary periods and $J_t$ is the number of secondary periods in primary period $t$: 

\begin{itemize}
    \item $y_{i, t, j}=1$ detected at the upper site
    \item $y_{i, t, j}=2$ detected at the lower site
    \item $y_{i, t, j}=3$ not detected
\end{itemize}

We use parameter-expanded data augmentation to account for the fact that the total number of adults in the population is unknown [@royle2012parameter]. 
Across the entire time period of the study, we assume $N_s$ unique individuals have been alive at either site. 
We observe $N$ unique individuals across all surveys, where $N \leq N_s$. 
An estimate of $N_s$ can be acquired by augmenting the observed capture histories with additional capture histories that consist entirely of non-detections, thus modeling a large number $M > N_s$ of individuals, $M - N_s$ of which never existed [@royle2009analysis]. 
Here, $M$ was chosen to be 2,182 (1,212 observed unique individuals plus 970 augmented individuals). 
We verified that posterior estimates of $N_s$ were much less $M$ to avoid problems on the boundary of this augmented parameter space [@dennis2015computational]. 

We denote the true state of individual $i$ in primary period $t$ as $s_{i, t}$. 
We assume that within a primary period, the state of each individual does not change. 
This assumption is justified by the short time intervals between secondary periods within primary periods.
Four states are possible: 

\begin{itemize}
    \item $s_{i, t} = 1$ alive at the upper site
    \item $s_{i, t} = 2$ alive at the lower site
    \item $s_{i, t} = 3$ not recruited
    \item $s_{i, t} = 4$ dead
\end{itemize}


### Observation model {-}

An emission matrix $\matr{\Omega}^{(i, t)}$ links observations to hidden states for individual $i$ in primary period $t$.
The rows in $\matr{\Omega}^{(i, t)}$ correspond to the state of individual $i$ in primary period $t$, and the columns correspond to observation probabilities such that the entry in the $m^{th}$ row, $n^{th}$ column is $Pr(y_{i, t, j} = n \mid s_{i, t} = m)$:

\[
\matr{\Omega}^{(i, t)} =
\begin{blockarray}{cccc}
  \text{Detected: upper} & \text{Detected: lower} & \text{Not detected}\\
\begin{block}{(ccc)c}
  p_{i, t} & 0 & 1 - p_{i, t} & \begin{tabular}{@{}c@{}}Alive: upper \end{tabular} \\
  0 & p_{i, t} & 1 - p_{i, t} & \begin{tabular}{@{}c@{}}Alive: lower \end{tabular} \\
  0 & 0 & 1 & \begin{tabular}{@{}c@{}}Not recruited \end{tabular} \\
  0 & 0 & 1 & \begin{tabular}{@{}c@{}}Dead \end{tabular} \\
\end{block}
\end{blockarray}
 \]

Here, $p_{i, t}$ is the probability of detection for individual $i$ if it is alive in primary period $t$. 
We allowed detection probabilities to vary as a function of expected Bd load [@joseph2018]:

$$\text{logit}(p_{i, t}) = \alpha_0 + \alpha_\text{g[i]}^{(g)} + \alpha_\text{g[i]}^{(z)} z_{i, t} + \epsilon^{(p)}_t,$$

where $\alpha_0$ is an intercept parameter, $\alpha_\text{g[i]}^{(g)}$ is an adjustment for group $g$, $g[i]$ is the group that individual $i$ belongs to, $\alpha_\text{g[i]}^{(z)}$ is the effect of Bd load on group $g$, $z_{i, t}$ is the Bd load of individual $i$ in primary period $t$, and $\epsilon^{(p)}_t$ is an adjustment on detection probability for primary period $t$.
Together these terms in the detection probability model account for differences among control, treated, and non-experimental animals, differential effects of Bd load on detection, and differences among primary periods. 

### State model {-}

The hidden states of each individual evolve as a Markov process with transition matrix $\matr{\Psi}^{(i, t)}$, where the element in the $m^{th}$ row, $n^{th}$ column is $\text{Pr}(s_{i, t+1} = n \mid s_{i, t} = m)$:
\[
\matr{\Psi}^{(i, t)} =
\begin{blockarray}{ccccc}
  \text{Alive: upper} & \text{Alive: lower} & \text{Not recruited} & \text{Dead} \\
\begin{block}{(cccc)c}
  \phi_{i, t} (1 - \nu^{(l)}) & \phi_{i, t} \nu^{(l)} & 0 & 1 - \phi_{i, t} & \begin{tabular}{@{}c@{}} Alive: upper \end{tabular} \\
  \phi_{i, t} \nu^{(u)} & \phi_{i, t} (1 - \nu^{(u)}) & 0 & 1 - \phi_{i, t} & \begin{tabular}{@{}c@{}} Alive: lower \end{tabular} \\
  \gamma_t \rho^{(u)} & \gamma_t (1 - \rho^{(u)}) & 1 - \gamma_t & 0 & \begin{tabular}{@{}c@{}} Not recruited \end{tabular} \\
  0 & 0 & 0 & 1 & \begin{tabular}{@{}c@{}} Dead \end{tabular} \\
\end{block}
\end{blockarray}
 \]

Here, $\phi_{i, t}$ is the probability of survival, $\nu^{(l)}$ and $\nu^{(u)}$ are the probabilities of moving to the lower or upper site respectively (conditional on survival), $\gamma_t$ is the probability of recruitment, and $\rho^{(u)}$ is the probability of recruiting into the upper site  conditional on recruitment. 

Survival probabilities were modeled as a function of Bd load: 

$$\text{logit}(\phi_{i, t}) = \beta_0 + \beta_\text{g[i]}^{(g)} + \beta_\text{g[i]}^{(z)} z_{i, t} + \epsilon^{(\phi)}_t,$$

where $\beta_0$ is an intercept parameter, $\beta_\text{g[i]}^{(g)}$ is an adjustment for group $g$, $g[i]$ is the group that individual $i$ belongs to, $\beta_\text{g[i]}^{(z)}$ is the effect of Bd load on group $g$, $z_{i, t}$ is the Bd load of individual $i$ in primary period $t$, and $\epsilon^{(\phi)}_t$ is an adjustment on detection probability for primary period $t$.
As with the detection model, these terms account for differences among groups, among individuals with different Bd loads, and among primary periods.

We treat the field experiment in 2015 as the first primary period, for which states are known for experimental animals. 
For example, if individual $i$ was captured and released at the upper site, then we know that $s_{i, t} = 1$. 
Initial states are not known for non-experimental animals, which could have been alive at either site (states 1 or 2), or in the not recruited class (state 3). 
Note that we are only interested in modeling the state and capture histories of animals that might have been alive. 
We assigned a Dirichlet(1, 1, 1) prior for the initial state distribution, which assigns equal prior density to each initial state.

### Bd load model {-}

We modeled Bd loads as being normally distributed on a transformed scale. 
Raw Bd loads were transformed using a $\log_{10}$ plus one transformation, then centered and scaled to have mean zero and unit standard deviation (in an attempt to avoid ill-conditioning, as expected Bd load is used in the detection and survival model components).
Let $z^{\text{obs}}_{i, t}$ represent the observed transformed Bd load, and $z_{i, t}$ represent the expected value for individual $i$ on primary period $t$.
The observation model for Bd loads of detected individuals was Gaussian on the transformed scale: 

$$z^{\text{obs}}_{i, t} \sim \text{Normal}(z_{i, t}, \sigma),$$

\noindent where $\sigma$ is an observation-level standard deviation parameter. 

Expected Bd load was modeled as a function of treatment, primary period, and individual identity: 

$$z_{i, t} = \mu_t + \beta_{\text{trt}[i]} + \epsilon_i,$$

\noindent where $\mu_t$ is a time-varying intercept, $\beta_{\text{trt}[i]}$ is an adjustment for the treatment group of individual $i$ (denoted $\text{trt}[i])$ to account for potential differences in mean Bd loads between treated, control, and non-experimental animals, and $\epsilon_i$ is an individual-level adjustment.

### Prior distributions {-}

We expected movement among sites to be rare, so both movement parameters ($\nu^{(l)}$ and $\nu^{(u)}$) were assigned Beta(2, 20) priors.
Primary period and individual-level adjustments were modeled using zero-mean normal distributions with unknown standard deviations specific to the process of interest, e.g., for the probability of entering the population: $[\lambda_{1:T}] = \prod_{t = 1}^{T} \text{Normal}(\lambda_t | 0, \sigma^{(\lambda)})$. 
Standard deviation parameters were given unit scale half normal priors, and all remaining parameters were given unit scale normal priors.

### Inference {-}

We sampled from the posterior distribution of this model using dynamic Hamiltonian Monte carlo in Stan. 
We drew 3000 iterations for each of four chains, using a maximum treedepth of 11 and an adapt_delta value of 0.99.
Convergence was checked by visual inspection of traceplots and with Rhat values, using Rhat < 1.1 as a threshold.
Models were fit using the rstan R package, version 2.19.3 [@rstan2020].

\clearpage

# Literature Cited {-}

<div id="refs"></div>

\clearpage

# Tables {-}

## Table S1 {-}

Characteristics of all sites used in the antifungal treatment experiments

\bigskip

```{r, include=FALSE}
t_1 <- read_csv(file = here::here("out", "tables", "expt_sites.csv"))
```
```{r tableS1, echo=FALSE}
kbl(
  t_1, col.names = c("Basin", "Site ID", "Experiment", "Life stage", "Category", "Elevation", "Depth", "Area"),
  booktabs = TRUE, linesep = "") %>%
  row_spec(0, bold = TRUE) %>%
  footnote(general = "Units for Elevation and Depth are meters and those for Area are m\\\\textsuperscript{2}", 
           general_title = "", escape = FALSE) %>%
  footnote(general = "Treatment = ``itracon + Jliv'' indicates the use of itraconazole and \\\\textit{J. lividum}", 
           general_title = "", escape = FALSE) %>%  
  footnote(general = "Life stage = ``early'' indicates tadpoles and subadults", general_title = "") %>%  
  kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S2 {-}

Timelines for antifungal treatment experiments conducted in the lower and upper basins of the LeConte study area in 2015.

(a) Lower basin
```{r, include=FALSE}
t_lower <- read_csv(file = here::here("out", "tables", "timeline-lower.csv")) 
```
```{r tableS2a, echo=FALSE}
kbl(t_lower, booktabs = TRUE, linesep = "") %>%
      row_spec(0, bold = TRUE) %>%
      column_spec(1, width = "7em") %>%
      column_spec(2, width = "7em") %>%
      column_spec(3, width = "7em") %>%
      column_spec(4, width = "7em") %>%
      column_spec(5, width = "7em") %>%
      column_spec(6, width = "7em") %>%
      column_spec(7, width = "7em") %>%
      kable_styling(latex_options = "scale_down")
```

(b) Upper basin
```{r, include=FALSE}
t_upper <- read_csv(file = here::here("out", "tables", "timeline-upper.csv")) 
```
```{r tableS2b, echo=FALSE}
kbl(t_upper, booktabs = TRUE, linesep = "") %>%
      row_spec(0, bold = TRUE) %>%
      column_spec(1, width = "7em") %>%
      column_spec(2, width = "7em") %>%
      column_spec(3, width = "7em") %>%
      column_spec(4, width = "7em") %>%
      column_spec(5, width = "7em") %>%
      column_spec(6, width = "7em") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S3 {-}

In for the itraconazole treatment experiments in Barrett and Dusy basins, results of model comparing Bd loads on frogs from ponds assigned to control and treated groups immediately before the treatment period. 

```{r, include=FALSE}
barrettdusy_m1 <- read_csv(file = here::here("out", "tables", "barrettdusy-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS3, echo=FALSE}
kbl(barrettdusy_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S4 {-}

For the itraconazole treatment experiment in Barrett and Dusy basins, results of model comparing Bd loads on frogs assigned to the treated group from before versus the end of the treatment period. 

```{r, include=FALSE}
barrettdusy_m2c <- read_csv(file = here::here("out", "tables", "barrettdusy-m2c.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS4, echo=FALSE}
kbl(barrettdusy_m2c, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 9) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is zero-inflated negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S5 {-}

For the Dusy Basin itraconazole treatment experiment, results of model comparing zoospore pools of ponds assigned to the control and treated groups before and after treatment.   

```{r, include=FALSE}
dusy2010_m1 <- read_csv(file = here::here("out", "tables", "dusy2010-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS5, echo=FALSE}
kbl(dusy2010_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Group-level effects", 1, 1) %>%
      pack_rows("Population-level effects", 2, 5) %>%
      pack_rows("Family-specific parameters", 6, 6) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S6 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs assigned to the control and treated categories immediately before the treatment period.   

```{r, include=FALSE}
leconte2015_m1 <- read_csv(file = here::here("out", "tables", "leconte2015-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS6, echo=FALSE}
kbl(leconte2015_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

## Table S7 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs assigned to the treated group from before versus the end of the treatment period. 

```{r, include=FALSE}
leconte2015_m2 <- read_csv(file = here::here("out", "tables", "leconte2015-m2.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS7, echo=FALSE}
kbl(leconte2015_m2, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S8 {-}

For the LeConte Basin itraconazole treatment experiment, results of model comparing Bd loads on frogs in the treated group that survived versus died.   

```{r, include=FALSE}
leconte2015_m4 <- read_csv(file = here::here("out", "tables", "leconte2015-m4.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS8, echo=FALSE}
kbl(leconte2015_m4, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is bernoulli", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

## Table S9 {-}

For the Treasure Lakes Basin itraconazole treatment, results of model comparing Bd loads on frogs before versus the end of the treatment period. 

```{r, include=FALSE}
treasure_m1 <- read_csv(file = here::here("out", "tables", "treasure-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS9, echo=FALSE}
kbl(treasure_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 3) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S10 {-}

For the Treasure Lakes Basin itraconazole treatment, results of model evaluating predictors of treatment effectiveness.

```{r, include=FALSE}
treasure_m2 <- read_csv(file = here::here("out", "tables", "treasure-m2.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS10, echo=FALSE}
kbl(treasure_m2, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 4) %>%
      pack_rows("Family-specific parameters", 5, 5) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is gaussian", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

## Table S11 {-}

For the Dusy Basin microbiome augmentation experiment, results of model comparing Bd loads on frogs assigned to the control and treated groups immediately before the itraconazole treatment period.   

```{r, include=FALSE}
dusy2012_m1 <- read_csv(file = here::here("out", "tables", "dusy2012-m1.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS11, echo=FALSE}
kbl(dusy2012_m1, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 3) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

## Table S12 {-}

For the Dusy Basin microbiome augmentation experiment, results of model comparing Bd loads on frogs assigned to the treated category from immediately before versus at the end of the itraconazole treatment.   

```{r, include=FALSE}
dusy2012_m2a <- read_csv(file = here::here("out", "tables", "dusy2012-m2a.csv")) %>% dplyr::mutate(Rhat = sprintf("%0.2f", Rhat))
```
```{r tableS12, echo=FALSE}
kbl(dusy2012_m2a, col.names = c(" ", "Estimate", "Est. Error", "lo95%CI", "up95%CI", "Rhat", "Bulk ESS", "Tail Ess"), align = "lrrrrrrr",
      booktabs = TRUE) %>%
      pack_rows("Population-level effects", 1, 2) %>%
      pack_rows("Family-specific parameters", 3, 4) %>%
      row_spec(0, bold = TRUE) %>%
      footnote(general = "Model family is zero-inflated negative binomial", general_title = "") %>%
      kable_styling(latex_options = "scale_down")
```

\clearpage

# Figures {-}

## Figure S1 {-}

```{r figa1, echo=FALSE, fig.cap=""}
knitr::include_graphics(here::here("out", "figures", "leconte-20052015-bdload.png"))
```

Temporal patterns of Bd infection in the LeConte study populations in the years prior to and the year of the treatment experiment. 
For the lower and upper basins of the study area, Bd prevalence (upper panels) and load (lower panels; on a log~10~ + 1 scale) during the period 2005-2015.
Estimates of average prevalence include Clopper-Pearson binomial 95% confidence intervals and sample sizes. Boxplot components are as in Figure 1. 
Hollow dots overlying the Bd load boxplots represent the actual values (jittered) used to calculate the box plots. Units for Bd load are the number of ITS1 copies.

\clearpage


